ghost/admin/app/components/gh-resource-select.js

```diff
@@ -146,4 +153,38 @@ export default class GhResourceSelect extends Component {
 
         this._options = options;
     }
+
+    @task({restartable: true})
+    *fetchOptionsForSearchTask(searchTerm) {
+        // Debounce
+        yield timeout(DEBOUNCE_MS);
+
+        const options = yield [];
+
+        if (this.args.type === 'email') {
+            const posts = yield this.store.query('post', {filter: '(status:published,status:sent)+newsletter_id:-null+title:~\'' + searchTerm.replace('\'', '\\\'') + '\'', limit: '10', fields: 'id,title'});
+            options.push(...posts.map(mapResource));
+            return;
```

Comment:
Don't we want to `return options` here?

Reply:
Oh my bad! Didn't spot this while testing